.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -66,7 +68,6 @@ body:
       - Keyboard Manager
       - Mouse Utilities
       - Mouse Without Borders
-      - New+ 
```

Comment:
why removing?

Reply:
@htcfreek Thank you for highlighting this!